inference-engine/src/CMakeLists.txt

```diff
@@ -34,6 +34,8 @@ add_subdirectory(inference_engine)
 
 add_subdirectory(low_precision_transformations)
 
+add_subdirectory(offline_transformations)
```

Comment:
Maybe we can change the structure of src folder and group all transformation libraries in one folder (like it already done for readers):
```
transformations:
    - offline
    - low_precision
    - common
```

Reply:
Make sense but I suggest to do it in a separate PR.